PrefectHQ/prefect
# Publish docs from release 2.7.11

- Implement non-blocking style pauses (#7738)
- Fix bug where pause expiration and late run services settings were swapped (#7847)
- Implement out-of-process flow run pauses (#7863)
- Refactor `pause_flow_run` for consistency with engine state handling (#7857)
- Add `WorkerThreadPool` for running synchronous work in threads (#7875)
- Bump @prefecthq/eslint-config from 1.0.19 to 1.0.20 in /orion-ui (#7921)
- Improve `Event` implementation consistency and fix deadlock (#7897)
- Bump @prefecthq/vue-compositions from 1.1.0 to 1.1.1 in /orion-ui (#7937)
- Remove worker pool in favor of reusing standard library executors (#7931)
- implement create can (#7963)
- Bump @prefecthq/vue-compositions from 1.1.1 to 1.1.3 in /orion-ui (#7987)
- Caps SQLAlchemy version below 2.0 and silences uber-warning (#8042)
- Workers pages and routes (#7973)
- add `validate` to `Block.load` enabling block document migration (#7862)
- Moves `SQLALCHEMY_SILENCE_UBER_WARNING` to setup.cfg (#8053)
- Adds worker pool and worker pool queue filters to count flow runs endpoint (#8043)
- Include worker pool name and worker pool queue name in flow run API responses (#8012)
- Allow schema migration of block documents during `Block.save` (#8056)
- Replace CodeBanner text with better temporary one (#8077)
- Extend `visit_collection` with support for annotations (#7263)
- Add `BaseWorker` and `ProcessWorker` (#7996)
- Rename Worker pools -> work pools (#8107)
- Bump @prefecthq/orion-design from 1.1.55 to 1.1.56 and @prefecthq/vue-compositions from 1.1.2 to 1.2.3 in /orion-ui and fix breaking changes (#8051)
- Add base and process worker config (#8100)
- Skip flaky test `test_dependent_events_in_two_loops_do_not_deadlock` (#8159)
- Chore: Bump orion and prefect design to 1.2.0 (#8163)
- Bump @prefecthq/vue-compositions from 1.2.3 to 1.2.9 in /orion-ui (#8175)
- Adds compatibility test for deployments (#8116)
- Latest orion design (#8219)
- Add `is_schedule_active` to client `Deployment` class (#7430)
- Adds migration logic for work pools (#8214)
- Removes migration logic (#8255)
- Create a `CANCELLING` state type (#7794)
- Reorder migrations for clarity
- Update comment for consistency
- Add Prefect Cloud Quickstart tutorial (#8227)
- Use standard warning handling for SQLAlchemy 2.0 (#8267)
- Configure docs to enable versioning at a future time (#8204)
- Update release notes example to run a command (#8278)
- Refactor results into a standalone artifact table (#8164)
- Add git commands to Prefect Recipes contribution page (#8283)
- Add retry_delay_seconds and exponential_backoff examples to Tasks retries documentation (#8280)
- Minor updates to the Docker image build workflow (#8279)
- Upgrade pinned pip version in Docker images (#8289)
- Adds ability to exclude experimental fields in API calls (#8274)
- Ensure that any preinstalled packages are upgraded in Docker images (#8288)
- Update issue label workflow (#8290)
- Update default task run name to exclude hash of task key (#8292)
- Bump eslint from 8.32.0 to 8.33.0 in /orion-ui (#8299)
- Bump @prefecthq/eslint-config from 1.0.22 to 1.0.23 in /orion-ui (#8300)
- Bump @prefecthq/prefect-design from 1.2.2 to 1.2.3 in /orion-ui (#8301)
- Bump @prefecthq/vue-compositions from 1.2.9 to 1.3.0 in /orion-ui (#8302)
- Bump @prefecthq/orion-design from 1.2.7 to 1.2.8 in /orion-ui (#8303)
- Add assertion for single notification and expand notification test suite (#7504)
- [maintenance] Update isort to fix errors in CI (#8298)
- Add role permissions regarding block secrets (#8309)
- Bump @playwright/test from 1.29.2 to 1.30.0 in /orion-ui (#8313)
- Remove unused Cloud Getting Started page (#8291)
- Fix Prefect Cloud typo in FAQ (#8317)
- Update Issue bot (#8297)
- Mention reverse proxy for PREFECT_API_URL config (#8240)
- Bump @prefecthq/orion-design from 1.2.8 to 1.2.9 in /orion-ui (#8326)
- Bump typescript from 4.9.4 to 4.9.5 in /orion-ui (#8325)
- Bump @prefecthq/vue-compositions from 1.3.0 to 1.3.3 in /orion-ui (#8328)
- allow snakecase in maps folder (#8322)
- Fix tests for scheduled flow runs CLI `--start-at` option (#8330)
- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logs outside of flow run context (#8311)
- Update issue label workflow (#8334)
- Expand `work_queue` table to accommodate work pools (#8264)
- Add getting started tutorial video to Prefect Cloud Quickstart (#8336)
- Add infra overrides tab to Deployments page (#8344)
- Work queue data migration (#8327)
- bump orion design to 1.2.12 (#8347)
- Bump @prefecthq/orion-design from 1.2.12 to 1.2.13 in /orion-ui (#8353)
- Enables updates to work queue `last_polled` time when polling work pools (#8338)
- Adds clean up migration (#8331)
- Fix cancellation bug (#8315)
- Bump @prefecthq/vue-compositions from 1.3.3 to 1.3.4 in /orion-ui (#8358)
- Add tips for re-registering blocks from Prefect Collections (#8333)
- Update CodeSnippet and fix odd issue with extra space on flowRun page (#8359)
- Introduces the Prefect Cloud Events schema and clients. (#8357)
- Chore: replaces CodeHighlighting with CodeSnippet (#8361)
- Bump @prefecthq/orion-design from 1.2.13 to 1.2.14 in /orion-ui (#8370)
- Examples and corrections for Kubernetes infrastructure overrides (#8312)
- Bump @prefecthq/orion-design from 1.2.14 to 1.2.15 in /orion-ui (#8375)
- Fix Kubernetes job watch exit when no timeout given (#8350)
- Fix environment variable name for 2.6 integration tests (#8376)
- Adds default value for priority on `WorkQueue` core schema (#8373)
- Bump @prefecthq/orion-design from 1.2.15 to 1.2.16 in /orion-ui (#8379)
- BugFix: Use recentFlowRunFilter instead of flowRunFilter (#8380)
- Add CLI support for work-pools (#8259)
- Update poolQueue code banner and remove unused code (#8381)
- Bump vue from 3.2.45 to 3.2.47 in /orion-ui (#8372)
- Bump vite from 4.0.4 to 4.1.1 in /orion-ui (#8369)
- Bump @prefecthq/prefect-design from 1.2.3 to 1.2.5 in /orion-ui (#8387)
- Bump @prefecthq/orion-design from 1.2.16 to 1.2.17 in /orion-ui (#8388)
- Resolves conflict when default work pool doesn't exist (#8384)
- Remove partial for Cloud link (#8389)
- Swap order of runs and deployments on flow page (#8386)
- Clearer output CLI message when switching profiles (#8383)
- Remove type requirement when creating a work pool via CLI (#8390)
- Add coverage for flow run with agent to integration test (#8374)
- Release Notes 2.7.11 (#8382)





### Example


### Checklist


	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
